src/react-app-env.d.ts

```diff
@@ -1 +1,21 @@
-/// <reference types="react-scripts" />
+export type User = {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+};
+
+export type Todo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+};
+
+export type TodosWithUsers = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null,
+};
```

Comment:
```suggestion
```
Move these types to separate files in `types` folder.
Revert original file content